aim/tests/unit/agent/aid_universes/test_converter.py

```diff
@@ -3836,37 +3836,62 @@ class TestAciToAimConverterBgpAsPPeer(TestAciToAimConverterBase,
     reverse_map_output = [
         {'resource': 'bgpLocalAsnP',
          'exceptions': {'local_asn': {
-                        'other': 'localAsn'}},
+             'other': 'localAsn'}},
          'skip': ['asn']},
         {'exceptions': {},
          'resource': 'bgpPeerP',
          'skip': ['asn', 'localAsn']},
         {'exceptions': {},
          'identity_converter': converter.bgp_as_id_converter,
-         'resource': 'bgpAsP',
+         'resource': 'bgpAsP__Peer',
          'skip': ['localAsn']}
     ]
-    sample_input = [[get_example_aci_bgppeerp(),
-                     get_example_aci_bgpaspeerp()],
-                    [get_example_aci_bgppeerp(tenant_name='t2'),
-                     _aci_obj('bgpAsP',
-                              dn='uni/tn-t2/out-l1/lnodep-np1/lifp-ip1/'
-                                 'rspathL3OutAtt-[topology/pod-1/paths-101/'
-                                 'pathep-[eth1/1]]/peerP-[1.1.1.0/24]/as',
-                              asn=0)]]
+    sample_input = [[
+        {'bgpPeerP': {
+         'attributes': {'rn': 'peerP-[1.1.1.0/24]',
+                        'dn': 'uni/tn-test_gbp/'
+                              'out-testOut1/lnodep-testNP1/lifp-testLifP1/'
+                              'rspathL3OutAtt-[topology/pod-1/paths-101/'
+                              'pathep-[eth1/1]]/peerP-[1.1.1.0/24]',
+                        'status': 'created'}, 'children': []}},
+        {'bgpAsP': {
```

Comment:
should we change this to 'bgpAsP__Peer' also then?

Reply:
Pls also add a UT to cover this bgpAsP__Peer case then.

Reply:
Sure, sorry I missed this. I will modify this case with another commit. The second part of this UT already has the regular bgpAsP case.